operatorcourier/validate.py

```diff
@@ -339,24 +343,23 @@ def _pkgs_validation(self, bundleData):
 
                     if "currentCSV" not in channel:
                         self._log_error("package channel.currentCSV not defined.")
-                    else:
-                        if channel["currentCSV"] not in csvNames:
-                            self._log_error("channel.currentCSV %s is not "
-                                            "included in list of csvs",
-                                            channel["currentCSV"])
-                            valid = False
+                    elif not nested and channel["currentCSV"] not in csvNames:
```

Comment:
This seems unnecessary and adds a lot of indirection to the method definitions. It's not really in the scope of the PR to add this check, right? Why add this?

Reply:
@kevinrizza Note that I only added `not nested` to the if condition. We need to remove this check for nested cases because `currentCSV` field in the package file can only match 1 CSV version. This will not work in nested cases where there are multiple CSVs. I tried to pass `nested` field inside also because of this.

Reply:
So if that's the case I think this needs more careful consideration -- perhaps a method that does nested verification on the package file separate from the non nested case. See my comments above.